distributionviewer/core/static/css/configuration.styl

```diff
@@ -0,0 +1,87 @@
+@import 'lib';
+
+.configuration {
+  margin-bottom: $med-padding;
+  max-height: 200px;
+  overflow-y: auto;
+  padding: $min-padding;
+
+  &[open] {
+    margin-bottom: $lrg-padding;
```

Comment:
I'll keep this here just to keep all the visibility stuff together. The compressor should combine them, thankfully.